docs/Writing_easyconfig_files.rst

```diff
@@ -112,6 +112,8 @@ Source files and patches
 * **sources**: list of source files (filenames only)
 * **source urls**: list of URLs where sources can be downloaded
 * **patches**: list of patch files to be applied (``.patch`` extension)
+* **checksums**: list of checksums (``adler32``, ``crc32``, ``md5`` (default), ``sha1``, ``size``)
+  to test the sources against
```

Comment:
not only sources, also patches

if multiple sources/patches are listed, the checksums are consumed in order (sources first), maybe that should be mentioned too?